sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -177,6 +178,36 @@ def test_fail_example(log_collector):
             raise ValueError('Did not stop executing script after error')
 
 
+class _FakeApp(object):
+    def __init__(self):
+        for key in ('info', 'warn', 'error', 'critical'):
+            setattr(self, key, lambda msg, **kwargs: logging.info(msg))
+
+    def status_iterator(self, iterable, *args, **kwargs):
+        for it in iterable:
+            yield it
+
+
+def test_gen_dir_rst():
+    """Test gen_dir_rst."""
+    _app = sphinx_compatibility._app
```

Comment:
It seems to work in my machine and travis is green. You can cherrypick from 
https://github.com/Titan-C/sphinx-gallery/commits/Eric89GXL/unicode